specificatie/openapi.yaml

```diff
@@ -648,6 +810,115 @@ components:
               items:
                 $ref: '#/components/schemas/ZoekResultaatHal'
 
+    GeoJsonZoekOpdracht:
+      description: 'Zoek opdracht voor het zoeken naar resource binnen een geometrie.'
+      type: object
+      required:
+        - geometrie
+        - spatialOperator
+      properties:      
+        geometrie:
+          description: 'De geometrie waarbinnen gezocht moet worden naar objecten. Alleen Point en Polygon worden ondersteund. Als Point wordt opgegeven, dan moet ook de paramter radius worden opgegeven zodat in een gebied met een straal rond het opgegeven punt wordt gezocht.' 
+          $ref: 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/geometryGeoJSON.yaml'
```

Comment:
deze definitie bevat oneOne. Dit geeft problemen bij het genereren. Kun je hier niet beter 
#/components/schemas/PuntOfVlak voor gebruiken?

Reply:
Ik heb nu een Geometrie type gedefinieerd die een cirkel van type Cirkel (allOf van point plus radius) of polygoon kan bevatten. Misschien niet de mooiste oplossing maar wel functioneel en levert geen code generatie problemen op. Evt. kan de cirkel nog gewijzigd worden naar: bevat point en radius.

Reply:
Ik heb ook een attribuut locatie van type Point toegevoegd, zodat er nu een keuze is tussen, punt, cirkel of polygoon.